Documentation/using-rkt-with-systemd.md

```diff
@@ -79,7 +81,17 @@ If the output contains `-XZ`, journal entries will not be available.
 
 ### Notifications
 
+Sometimes, defining dependencies between containers makes sense.
+An example use case of this is a container running a server, and another container running a client.
+We want the server to start before the client tries to connect.
+
+![sd_notify-background](sd_notify-background.svg)
```

Comment:
The image looks huge on [here](https://github.com/kinvolk/rkt/blob/b90939f9549bab3151766be504f36581ebf785d8/Documentation/using-rkt-with-systemd.md). Could it be a bit smaller?

Reply:
I couldn't find an easy way to adjust the size (apart from using html with the image URL) and I don't think it's so huge. I'd keep everything like it is.